core/controlplane/config/templates/cloud-config-worker

```diff
@@ -142,7 +142,8 @@ coreos:
         {{end}}--register-node=true \
         {{if .Experimental.Taints}}--register-schedulable=false \
         {{end}}--allow-privileged=true \
-        --pod-manifest-path=/etc/kubernetes/manifests \
+        {{if .Experimental.NodeStatusUpdateFrequency}}--node-status-update-frequency={{.Experimental.NodeStatusUpdateFrequency}}
```

Comment:
I know I need to get the setting into the node pools... just not quite sure how.  Any example setting that does a similar thing that you could point me at? @mumoshu ?